pkg/config/debug.go

```diff
@@ -60,4 +60,6 @@ type PullerConfig struct {
 	EnableResolvedTsStuckDetection bool `toml:"enable-resolved-ts-stuck-detection" json:"enable-resolved-ts-stuck-detection"`
 	// ResolvedTsStuckInterval is the interval of checking resolved ts stuck.
 	ResolvedTsStuckInterval TomlDuration `toml:"resolved-ts-stuck-interval" json:"resolved-ts-stuck-interval"`
+	// LogRegionDetails determines whether logs Region details or not in puller and kv-client.
```

Comment:
Perhaps it would be more appropriate to use changefeed level parameters and support online updates for LogRegionDetails?

Reply:
Just a server level config is fine enough.

Reply:
@hicqu @CharlesCheung96 TiCDC has already supported online log level changing. So we can change log level to debug level to print these logs when needed.